src/index.html

```diff
@@ -9,6 +9,8 @@
   <link rel="stylesheet" href="style.css">
 </head>
 <body>
-  <h1>Colored blocks</h1>
+  <div class="r h"></div>
+  <div class="g h"></div>
+  <div class="b h"></div>
```

Comment:
how do you think other developers know what you mean by those names?

you should give clear and descriptive names